.vscode/settings.json

```diff
@@ -5,4 +5,7 @@
     "source.fixAll": true,
     "prettier.singleQuote": true,
   },
+  "eslint.workingDirectories": [
+    "./client", "./server"
+  ]
 }
```

Comment:
remove this line from here, we said you can add the setting from vs code setting itself not from here.
just open vs code setting then search for a file called `settings.json` then add your work directories in it .